model/registry/aws.yaml

```diff
@@ -240,3 +240,117 @@ groups:
                   "WriteCapacityUnits": number
                 }
               }'
+  - id: registry.aws.ecs
```

Comment:
I think this is similar to the https://github.com/open-telemetry/semantic-conventions/pull/872#discussion_r1549913202

Reply:
I created an issue to keep the discussion going: https://github.com/open-telemetry/semantic-conventions/issues/887